recipes/backward-cpp/all/conanfile.py

```diff
@@ -0,0 +1,152 @@
+            if self.options.stack_details_bfd:
+                required_package = "binutils-dev"
+        
+        if required_package != None:
+            installer = tools.SystemPackageTool()
+            if not installer.installed(required_package):
+                raise ConanInvalidConfiguration("backward-cpp requires {}.".format(required_package))
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.definitions['STACK_WALKING_UNWIND'] = self.options.stack_walking_unwind
+        cmake.definitions['STACK_WALKING_BACKTRACE'] = self.options.stack_walking_backtrace
+        cmake.definitions['STACK_DETAILS_AUTO_DETECT'] = self.options.stack_details_auto_detect
+        cmake.definitions['STACK_DETAILS_BACKTRACE_SYMBOL'] = self.options.stack_details_backtrace_symbol
+        cmake.definitions['STACK_DETAILS_DW'] = self.options.stack_details_dw
+        cmake.definitions['STACK_DETAILS_BFD'] = self.options.stack_details_bfd
+        cmake.definitions['STACK_DETAILS_DWARF'] = self.options.stack_details_dwarf
+        cmake.definitions['BACKWARD_SHARED'] = self.options.shared
+        cmake.configure(build_folder=self._build_subfolder)
+        return cmake
+
+    def build(self):
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        cmake = self._configure_cmake()
+        cmake.install()
+        self.copy(pattern="LICENSE*", dst="licenses", src=self._source_subfolder)
+        os.remove(os.path.join(self.package_folder, "lib", "backward", "BackwardConfig.cmake"))
+
+    def package_info(self):
+        self.cpp_info.names["cmake"] = "Backward"
+        self.cpp_info.names["cmake_find_package"] = "Backward"
+        self.cpp_info.names["cmake_find_package_multi"] = "Backward"
+
+        self.cpp_info.defines.append('BACKTRACE_HAS_UNWIND={}'.format(int(self.options.stack_walking_unwind == True)))
+        self.cpp_info.defines.append('BACKTRACE_HAS_BACKTRACE={}'.format(int(self.options.stack_walking_backtrace == True)))
+        
+        self.cpp_info.defines.append('BACKTRACE_HAS_BACKTRACE_SYMBOL={}'.format(int(self.options.stack_details_backtrace_symbol == True)))
+        self.cpp_info.defines.append('BACKTRACE_HAS_DW={}'.format(int(self.options.stack_details_dw == True)))
+        self.cpp_info.defines.append('BACKTRACE_HAS_BFD={}'.format(int(self.options.stack_details_bfd == True)))
+        self.cpp_info.defines.append('BACKTRACE_HAS_DWARF={}'.format(int(self.options.stack_details_dwarf == True)))
+
+        self.cpp_info.libs = tools.collect_libs(self)
+        if self.settings.os == "Linux":
+            self.cpp_info.libs.extend(["dl"])
```

Comment:
This is a system library, right?
```suggestion
            self.cpp_info.system_libs.extend(["dl"])
```